### PR TITLE
Require Django >= 2.1 and Python >= 3.6

### PR DESCRIPTION
**Pull request by dyve**
Dropped support for older versions.
Went back to tox-travis.
Prepare release 12.0.0

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,13 @@
+dist: xenial
+sudo: false
 language: python
-cache:
-  pip: true
 addons:
-  postgresql: 9.6
+  postgresql: "9.6"
```
Comment (jieter) [inline .travis.yml]: might be worthwhile to also upgrade postgresql too?

File: bootstrap3/forms.py [hunk 2]
```diff
@@ -120,10 +106,7 @@ def render_button(
     elif size == "md" or size == "medium":
         pass
     elif size:
-        raise BootstrapError(
-            'Parameter "size" should be "xs", "sm", "lg" or '
-            + 'empty ("{}" given).'.format(size)
-        )
+        raise BootstrapError('Parameter "size" should be "xs", "sm", "lg" or ' + 'empty ("{}" given).'.format(size))
```
Comment (jieter) [inline bootstrap3/forms.py]: `+` seems to be an artefact of the previous formatting.

File: setup.py [hunk 3]
```diff
@@ -28,6 +25,14 @@
...
     sys.exit()
 
+if sys.argv[-1] == "reformat":
+    os.system("isort -rc bootstrap3")
+    os.system("isort -rc demo")
+    os.system("autoflake -ir bootstrap3 demo --remove-all-unused-imports")
+    os.system("black .")
+    os.system("flake8 bootstrap3 demo")
+    sys.exit()
+
```
Comment (jieter) [inline setup.py]: this also ends in user installations of the package (not really a problem, but slightly messy). You might want to consider moving the 'build'-related code to separate script

for example: https://github.com/jieter/django-tables2/blob/master/maintenance.py

File: .travis.yml (hunk 1 above)
Comment (dyve) [reply .travis.yml]: I think 9.6 is xenial default, and we only need very little Postgres, so I' leave it like this for now.

File: setup.py (hunk 3 above)
Comment (dyve) [reply setup.py]: In that case why not a Makefile or fabfile? A python script that does `os.sys` calls seems like the wrong tool for the job (but it gets it done).

**Comment by dyve** [comment]
I'll check for Python 2 specific code such as `super` calls.

File: setup.py (hunk 3 above)
Comment (dyve) [reply setup.py]: I turned it into a Makefile

**Pull request closed without merging**